tensorflow/contrib/seastar/seastar_tensor_coding.cc

```diff
@@ -0,0 +1,38 @@
+
+#include <pthread.h>
```

Comment:
Same above. Are you using these Linux specific headers?

Reply:
i suppose, these code is useless.